doc/nrf/doc_structure.rst

```diff
@@ -0,0 +1,55 @@
+.. _doc_structure:
+.. _documentation_sets:
+
+Documentation structure
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. doc_structure_start
+
+The documentation consists of several inter-linked documentation sets, one for each repository.
+
+The entry point is the |NCS| documentation that you are currently reading.
+The local :doc:`Zephyr documentation <zephyr:index>` is a slightly extended version of the official `Zephyr Project documentation`_, containing some Nordic Semiconductor specific additions.
+The local :doc:`MCUboot documentation <mcuboot:index>` is a slightly extended version of the official `MCUboot`_ documentation, containing some Nordic Semiconductor specific additions.
+
+You can switch between these documentation sets by using the selector in the *bottom-left corner of each documentation page*.
+
+.. figure:: images/switcher_docset_snipped.gif
+   :alt: nRF Connect SDK documentation set selector
+
+   |NCS| documentation set selector
+
+.. doc_structure_end
+
+Following are the available documentation sets:
+
+- ``nrf``: |NCS|
+- ``nrfx``: nrfx
+- ``nrfxlib``: nrfxlib
+- ``zephyr``: Zephyr RTOS
+- ``mcuboot``: MCUboot
+- ``tfm``: Trusted Firmware-M
+- ``matter``: Matter
+- ``kconfig``: All available Kconfig options in the |NCS|
+
+Since there are links from the |NCS| documentation set into other documentation sets, the documentation is built in a predefined order, also when you :ref:`build it manually <doc_build_steps>`.
+
+.. _doc_structure_versions:
+
+Different versions
```

Comment:
I can't think of a better title yet, but it might be good to be more specific with this title.
Maybe "Different documentation versions" or "Documentation set versions"?